internal/exec/stages/disks/filesystems.go

```diff
@@ -91,18 +91,19 @@ func (s stage) createFilesystem(fs types.Filesystem) error {
 	if fs.Format == nil {
 		return nil
 	}
+	devAlias := util.DeviceAlias(string(fs.Device))
```

Comment:
I believe you this fixes it but honestly I hadn't looked at `device_alias.go` before...would it be fair to say this is basically creating a "snapshot" of the device symlinks at a point in time so they're stable?  (OK yes, a quick blame takes me to 521b947706a7a708c366b2a1bdd8ff418a1abe83 - though that would be a good comment at the top of the file)

Reply:
Yup, it's a recurring pattern throughout the disks stage.